### PR TITLE
[544435] regenerated test languages

### PR DESCRIPTION
**Pull request by cdietrich**
[544435] regenerated test languages
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=544435

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=544435

File: org.eclipse.xtext.xbase.testlanguages/src-gen/org/eclipse/xtext/xbase/testlanguages/bug462047/serializer/Bug462047LangSyntacticSequencer.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ else if (ruleCall.getRule() == grammarAccess.getOpSingleAssignRule())
 	protected String getArrayBracketsToken(EObject semanticObject, RuleCall ruleCall, INode node) {
 		if (node != null)
 			return getTokenText(node);
-		return "[]";
+		return "[ ]";
```
Comment (kthoms) [inline org.eclipse.xtext.xbase.testlanguages/src-gen/org/eclipse/xtext/xbase/testlanguages/bug462047/serializer/Bug462047LangSyntacticSequencer.java]: are these spaces intended?

Comment (cdietrich) [reply org.eclipse.xtext.xbase.testlanguages/src-gen/org/eclipse/xtext/xbase/testlanguages/bug462047/serializer/Bug462047LangSyntacticSequencer.java]: for now yes

**Pull request merged**